### DISCUSSION TITLE
Trouble Opening Comfy UI

### DISCUSSION THREAD

**Discussion by johnny5924** [comfyanonymous/ComfyUI#5332] [General]
I double click my cpu.batt file and the response is an error message (I think) and tells me to press any key and when I do this the message disappears. The folly is the message I keep getting:

F:\Comfy UI\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --cpu --windows-standalone-build
[START] Security scan
WARNING: Ignoring invalid distribution ~orch (F:\Comfy UI\ComfyUI_windows_portable\python_embeded\Lib\site-packages)
[DONE] Security scan
Failed to execute startup-script: F:\Comfy UI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py / Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet

Prestartup times for custom nodes:
   0.8 seconds (PRESTARTUP FAILED): F:\Comfy UI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Traceback (most recent call last):
  File "F:\Comfy UI\ComfyUI_windows_portable\ComfyUI\main.py", line 88, in <module>
    import comfy.utils
  File "F:\Comfy UI\ComfyUI_windows_portable\ComfyUI\comfy\utils.py", line 20, in <module>
    import torch
  File "F:\Comfy UI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\__init__.py", line 262, in <module>
    _load_dll_libraries()
  File "F:\Comfy UI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\__init__.py", line 258, in _load_dll_libraries
    raise err
OSError: [WinError 126] The specified module could not be found. Error loading "F:\Comfy UI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\lib\caffe2_nvrtc.dll" or one of its dependencies.

F:\Comfy UI\ComfyUI_windows_portable>pause
Press any key to continue . . .

Pls point me in the right direction if this is not appropriate forum to make inquiry. 

John Perez

AMD Ryzen 7 3700 8-Core Processor  3.60 GHz
Installed RAM 32 GB
64-bit operating system x64-based Processor 
Windows 10 Home

**Comment by ltdrdata** [comment]
See this for caffe2_nvrtc.dll issue.
https://discuss.pytorch.org/t/cannot-import-torch-error-loading-caffe2-nvrtc-dll/166808


Btw you need to install git client.
Install standalone version from here:
https://git-scm.com/downloads/win
